app/main.py

```diff
@@ -1,25 +1,73 @@
+from __future__ import annotations
 from abc import ABC
+from typing import Union, Type
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(
+            self,
+            instance: Visitor,
+            owner: Type[Visitor]
+    ) -> Union[str, int]:
```

Comment:
```suggestion
    ) ->Optional[Union[str, int]]:
```

Reply:
I understood that's much better.
Added return annotation with Optional.